pkgs/games/owmods-cli/default.nix

```diff
@@ -6,30 +6,36 @@
 , pkg-config
 , installShellFiles
 , zstd
+, libsoup
+, makeWrapper
+, mono
+, wrapWithMono ? true
 , openssl
 , Security
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "owmods-cli";
-  version = "0.12.2";
+  version = "0.13.0";
 
   src = fetchFromGitHub {
     owner = "ow-mods";
     repo = "ow-mod-man";
     rev = "cli_v${version}";
-    hash = "sha256-AfqpLL3cGZLKW5/BE6SaBe4S8GzYM2GKUZU8mFH5uX4=";
+    hash = "sha256-JCPuKGO0pbhQaNmZUcZ95EZbXubrjZnw0qJmKCGuAoQ=";
   };
 
-  cargoHash = "sha256-PhdfpiUgeOB13ROgzPBYM+sBLGMP+RtV9j9ebo8PpJU=";
+  doCheck = false;
```

Comment:
Why disabling the entire check instead of dropping some test cases?

Reply:
Not really sure why it was set to false to be honest, but allowing it to be true isn't resulting in any issues, so I guess there isn't a problem in removing that line.